fixtures/iteration/python/leap/long-utf8.py

```diff
@@ -0,0 +1,28 @@
+# The first 1024 bytes of this file need to contain only ASCII characters.
```

Comment:
Of two minds. On the one hand, this is a test that knows a little too much about the implementation of the code it's testing (1024 bytes).

On the other hand, this is a real thing that happened and mangled my submission and someone else's, so it is not unwarranted.